### PR TITLE
Setup a miniDREAM RStudio test public env (public subnet + exposed port).

### PR DESCRIPTION
**Pull request by milen-sage**

### CODE REVIEW

File: config/prod/minidream-R-env-pub-instance-ec2.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v5.yaml
+stack_name: minidream-R-env-pub-instance2
```
Comment (zaro0508) [inline config/prod/minidream-R-env-pub-instance-ec2.yaml]: would you mind changing this to match the file name?

File: config/prod/minidream-R-env-test-instance-ec2.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Provision an EC2 instance
-template_path: managed-ec2-v4.yaml
+template_path: templates/managed-ec2-v4.yaml
```
Comment (zaro0508) [inline config/prod/minidream-R-env-test-instance-ec2.yaml]: this should not have changed

File: config/prod/minidream-R-env-pub-instance-ec2.yaml (hunk 1 above)
Comment (milen-sage) [reply config/prod/minidream-R-env-pub-instance-ec2.yaml]: Done.

File: config/prod/minidream-R-env-test-instance-ec2.yaml (hunk 2 above)
Comment (milen-sage) [reply config/prod/minidream-R-env-test-instance-ec2.yaml]: Done.

**Pull request merged**